lib/data/entity.js

```diff
@@ -41,14 +41,13 @@ const normalizeUuid = (id) => {
 };
 
 // Input: object representing entity, parsed from submission XML
-const extractLabelFromSubmission = (entity) => {
-  const { label, create, update } = entity.system;
-  if ((create === '1' || create === 'true') && (!label || label.trim() === ''))
-    throw Problem.user.missingParameter({ field: 'label' });
-
-  if ((update === '1' || update === 'true') && (!label || label.trim() === ''))
+const extractLabelFromSubmission = (entity, options = { create: true }) => {
+  const { create, update } = options;
```

Comment:
@matthew-white I added this options arg because I was running into a problem where a submission specified both `create` and `update`, but the validation behavior needed to be different depending on if we were updating or creating.